user_guide_src/source/libraries/uri.rst

```diff
@@ -234,6 +234,19 @@ what the values of the segments are. The segments start at 1 being the furthest
 		echo $request->uri->getSegment(2);
 	}
 
+You can also get the different default value for a given segment. Default value is set by using the second parameter of the ``getSegment()`` method.
+::
+	// URI = http://example.com/users/15/profile
+
+	// will print 'profile'
+	echo $request->uri->getSegment(3, 'foo');
```

Comment:
Change into $uri->, also change the rest of the code (that's not yours in URI Segments) into the same layout for consistency.

Reply:
Done.